### PR TITLE
Update cross-compile-arm.md

### PR DESCRIPTION
**Pull request by camillobruni**

### CODE REVIEW

File: src/docs/cross-compile-arm.md [hunk 1]
```diff
@@ -4,16 +4,20 @@ description: 'This document explains how to cross-compile V8 for ARM/Android, an
 ---
 First, make sure you can [build with GN](/docs/build-gn).
 
-Then, add `android` to your `.gclient` configuration file.
+Then, add `android` to your `.gclient` configuration file to get the neceasseray android dependencies checked out:
```
Comment (alexanderschulze) [inline src/docs/cross-compile-arm.md:7]: type

File: src/docs/cross-compile-arm.md [hunk 2]
```diff
@@ -4,16 +4,20 @@ description: 'This document explains how to cross-compile V8 for ARM/Android, an
...
-target_os = ['android']  # Add this to get Android stuff checked out.
+```
+...
+"target_os:" ["android"],
+...
 ```
 
 The `target_os` field is a list, so if you're also building on unix it'll look like this:
 
 ```python
```
Comment (alexanderschulze) [inline src/docs/cross-compile-arm.md:17]: Maybe consistently use the python syntax or remove it completely?

File: src/docs/cross-compile-arm.md [hunk 3]
```diff
@@ -4,16 +4,20 @@ description: 'This document explains how to cross-compile V8 for ARM/Android, an
 ---
 First, make sure you can [build with GN](/docs/build-gn).
 
-Then, add `android` to your `.gclient` configuration file.
+Then, add `android` to your `.gclient` configuration file to get the neceasseray android dependencies checked out:
 
-```python
-target_os = ['android']  # Add this to get Android stuff checked out.
+```
```
Comment (alexanderschulze) [inline src/docs/cross-compile-arm.md:9]: Maybe consistently use the python syntax or remove it from all snippets?

File: src/docs/cross-compile-arm.md (hunk 1 above)
Comment (camillobruni) [reply src/docs/cross-compile-arm.md:7]: I misread this, updating again

**Pull request closed without merging**